src/containers/LandingPageContainer.jsx

```diff
@@ -5,11 +5,13 @@ import LandingPageComponent from '../components/LandingPageComponent'
 import { goToStep } from '../actions/navigationActions'
 import { getTransferHistory } from '../actions/transferActions'
 import { createLoadingSelector, createErrorSelector } from '../selectors'
+import { getCloudWallet } from '../actions/walletActions'
 
 class LandingPageContainer extends Component {
   componentDidMount () {
     if (this.props.profile.isAuthenticated) {
       // load transfer history for logged-in users
+      this.props.getCloudWallet()
```

Comment:
This step is time consuming, check if cloudWalletConnected == true before calling getCloudWallet()

Reply:
Also fix the comment above line 14